ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -298,12 +301,24 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
         }
         let capabilities = capabilitiesStr.compactMap { Capability(rawValue: $0) }
 
+        var forwardJumpInterval: NSNumber = lastForwardJumpInterval;
+        if let explicitForwardJumpInterval = options["forwardJumpInterval"] as? NSNumber {
+            forwardJumpInterval = explicitForwardJumpInterval;
+            lastForwardJumpInterval = forwardJumpInterval;
+        }
+
+        var backwardJumpInterval: NSNumber = lastBackwardJumpInterval;
+        if let explicitBackwardJumpInterval = options["backwardJumpInterval"] as? NSNumber {
+            backwardJumpInterval = explicitBackwardJumpInterval;
+            lastBackwardJumpInterval = backwardJumpInterval;
+        }
+
         player.remoteCommands = capabilities.map { capability in
-            capability.mapToPlayerCommand(forwardJumpInterval: options["forwardJumpInterval"] as? NSNumber,
-                                          backwardJumpInterval: options["backwardJumpInterval"] as? NSNumber,
-                                          likeOptions: options["likeOptions"] as? [String: Any],
-                                          dislikeOptions: options["dislikeOptions"] as? [String: Any],
-                                          bookmarkOptions: options["bookmarkOptions"] as? [String: Any])
+            capability.mapToPlayerCommand(forwardJumpInterval: forwardJumpInterval,
+                                            backwardJumpInterval: backwardJumpInterval,
```

Comment:
@fivecar here I am being nitpicky, but I'd really prefer not to co-mingle simple stylistic changes (like the additional whitespace here) with fixes.

Reply:
Agreed - I shouldn't have done this. Reverted!